### PR TITLE
Update repo structure description, fix links

### PR DESCRIPTION
**Pull request by mark-moir**
Signed-off-by: Mark Moir <mark.moir@oracle.com>

### CODE REVIEW

**Comment by mark-moir** [comment]
I self-approve!

File: README.md [hunk 1]
```diff
@@ -35,13 +46,13 @@ To work with `Agda-LBFT`, you need to have Agda and its standard library install
...
 
-To explore the repo, we suggest starting with the [`LibraBFT.Abstract.Properties`](LibraBFT/Abstract/Properties.agda) module if you are interested in exploring the abstract correctness proof, and with the [`LibraBFT.Yasm.Properties`](LibraBFT/Yasm/Properties.agda) module if you are interested in the system model and its properties.
+To explore the repo, we suggest starting with the [`LibraBFT.Abstract.Properties`](src/LibraBFT/Abstract/Properties.agda) module if you are interested in exploring the abstract correctness proof, and with the [`LibraBFT.Yasm.Properties`](src/LibraBFT/Yasm/Properties.agda) module if you are interested in the system model and its properties.
 
 If you would like to consider contributing to the project, please see our [Contribution Guide](CONTRIBUTING.md).
 
 ## Get Support
 
-* Open a [GitHub issue](https://github.com/oracle/bft-consensus-agda/issues) to provide feedback, raise issues and ask questions.
+* Open a [GitHub issue](../../issues) to provide feedback, raise issues and ask questions.
```
Comment (haroldcarr) [inline README.md:55]: GOOD CATCH!

Comment (haroldcarr) [reply README.md:55]: BUT WAIT - this PR is for main - not for the blind repo.   It should not change this.   That should only be done in the FM brannch.

**Review by haroldcarr — APPROVED** [review]
Approved - but I suggest putting that `issues` link back.

**Pull request merged**